src/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -364,6 +364,13 @@ public static unsafe ushort RhGetComponentSize(EETypePtr ptrEEType)
             return pEEType->ComponentSize;
         }
 
+        [RuntimeExport("RhGetBaseSize")]
+        public static unsafe uint RhGetBaseSize(EETypePtr ptrEEType)
```

Comment:
I agree ... the nice side effect of this would be elimination of the call overhead for these trivial assessors. It makes me sad every time I step through one of those.